services/spar/schema/src/V5.hs

```diff
@@ -0,0 +1,18 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes       #-}
+
+module V5 (migration) where
+
+import Imports
+import Cassandra.Schema
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 5 "Store SCIM user blobs" $ do
+    void $ schema' [r|
+        CREATE TABLE if not exists scim_user
+            ( uid    uuid
```

Comment:
Oh, except for another place in Spar. And we can't rename columns that are not part of the primary key.

I guess we should leave this one in for consistency, but it's unpleasant that we'll have to live with it ("similar things have similar names" is nice because at least I use grepping to find bugs that the compiler doesn't catch).